Bond/Bond+Arrays.swift

```diff
@@ -82,6 +86,12 @@ public class DynamicArray<T>: Dynamic<Array<T>>, SequenceType {
     bond.bind(self, fire: fire, strongly: strongly)
   }
   
+  public override var value: Array<T> {
+    didSet {
+      dynCount.value = count
+    }
+  }
+  
```

Comment:
Well, `DynamicArray` is a subclass of `Dynamic` so from that point of view it's OK. In practice, there should be no reason to use this functionality.